docs/developer-docs/functionality/sns/deployment.md

```diff
@@ -1 +1,138 @@
-# SNS Deployment Runbook
\ No newline at end of file
+# Launching the SNS in production
+This describes the detailed steps for launching an SNS.
+
+:::caution
+
+In the following steps we assume that you have already
+[collected the developer and airdrop principals and chosen
+the initial SNS parameters in a .yaml file](predeployment.md).
+We assume that you have one developer principal that is compatible
+with both dfx and SNS quill <!-- TODO: explain what this means and link to where
+explained--> that you will use for the following steps.
+We also recommend that before following these steps and launch an
+SNS in production, you have [tested the SNS](local-testing.md).
+
+:::
+
+You are now ready to launch your SNS. There are some steps that are needed
+for launching an SNS and some steps that we recommend as intermediate
+sanity checks that up to this point everything works fine.
+We mark the latter by the keyword _(Recommended)_.
+
+#### 1. Make a call to the SNS wasm modules canister on the NNS subnet.
+To make this call, use the command as described 'here'. <!-- TODO: add link -->
+Upon receiving this call, the SNS wasm modules cansiter will deploy
+an SNS with your chosen initial parameters.
+
+
+#### 2. Add the SNS root canister as a controller to your dapp canister(s).
+To do so, use the command as described 'here'.
+<!-- TODO: add this to CLI/dfx tool as need to learn SNS canisters -->
+
+
+#### 3. _(Recommended)_ Test upgrading the dapp canister(s) by SNS proposal. {#step3}
+To test this, make an SNS proposal to upgrade one of the dapp canisters to
+a new wasm version.
+Then, ensure that sufficiently many initial neurons (deveoper and airdrop
+neurons) vote on the proposal so that it is adopted.
+In this way, confirm that the proposal has been upgraded. 
```

Comment:
```suggestion
In this way, confirm that the proposal has been executed. 
```
or
```suggestion
In this way, confirm that the wasm has been upgraded. 
```

Reply:
Added a bit more information in the next commit